styles/main.css

```diff
@@ -0,0 +1,199 @@
+body {
+  font-family: sans-serif;
+  width: 98%;
```

Comment:
Ooo smart, I didn't think of this! It's a clever way to make sure the border lines don't meet the edges of the page. It does mean the background color doesn't meet the edges. I think there must be a way to isolate this within the border, we'll have to look into it.